behat.yml.dist

```diff
@@ -6,11 +6,6 @@ default:
     extensions:
         DMore\ChromeExtension\Behat\ServiceContainer\ChromeExtension: ~
 
-        Lakion\Behat\MinkDebugExtension:
-            directory: etc/build
-            clean_start: false
-            screenshot: true
```

Comment:
Shouldn't we change to use `FriendsOfBehat\MinkDebugExtension` instead of removing this part of config?

Reply:
Both ways will work, if this lines are not here they are taken from vendor/sylius.